lib/virt_feature_test_base.pm

```diff
@@ -179,4 +180,17 @@ sub get_free_mem {
     }
 }
 
+sub get_pool_available_space {
```

Comment:
Better to name it get_active_pool_and_available_space, and return both active_pool_name and available size.

Reply:
sure, had changed as `get_active_pool_and_available_space(),` and return both `$active_pool_name` and `$available_size` now.